docs/docs.polserver.com/pol100/include/escriptguide.inc

```diff
@@ -255,7 +255,7 @@ eScript are:</p>
   </tr>
 </table>
 
-<h3><b>Assign, Unary, and Logical Operators</b></h3>
+<h3><b>Assign, Unary, Binary and Bitwise Operators</b></h3>
```

Comment:
A "binary operator" is any operator that takes two operands.  So multiplication, addition, logical or/and, etc.  (Most operators)
```suggestion
<h3><b>Assign, Unary, and Bitwise Operators</b></h3>
```